### DISCUSSION TITLE
Passport Configuration with MS SQL SERVER

### DISCUSSION THREAD

**Discussion by HenryMetarom** [laravel/framework#43493] [Q&A]
- Laravel Version : Laravel Framework 9.19.0
- PHP Version : PHP 8.1.4
- Database Driver :  sqlsrv
- Microsoft SQL Server 2019 (RTM-GDR) (KB4583458) - 15.0.2080.9 
- Collation : FRENCH_CI_AS

The `php artisan passport:client`  or `php artisan passport:install --uuids` command can't work after the 12 of the month.
I got an SQL STATE Error because SQL SERVER inverts days and months, so the auto generate date by Passport is impossible to read.

I know we can override default models but I have no idea about how to do this.

I can't reproduce errors message right now because of the date.

**Comment by kaozaza2** [comment]
Try [https://stackoverflow.com/a/36982138](https://stackoverflow.com/a/36982138)
